packages/server/src/channels/twilio/channel.ts

```diff
@@ -37,7 +37,8 @@ export class TwilioChannel extends Channel<TwilioConduit> {
           await this.app.instances.receive(conduit.conduitId, req.body)
           res.sendStatus(204)
         } else {
-          this.logger.error('Request validation failed. Make sure that your authToken is valid.')
+          // Maybe throw a exception here. Or just do new Error() and log that?
+          this.logger.error(undefined, 'Request validation failed. Make sure that your authToken is valid')
```

Comment:
I think creating a new error is fine. It will require less code than throwing an exception and is still easy to read!